.classpath

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Do we really need to version this file? if not, let's add it to `.gitignore`

Reply:
Added to .gitignore